tmail-backend/jmap/extensions/pom.xml

```diff
@@ -62,6 +62,10 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
+            <artifactId>apache-james-backends-es-v7</artifactId>
+        </dependency>
```

Comment:
We need a separate maven module for JMAP + ES dependency.

Reply:
It's still not  in a separated module

Reply:
Your mean is move source code (_EmailAddressContactFactory, ESEmailAddressContactSearchEngine_) to `apache-james-backends-es-v7`?

Reply:
We mean "don''t put ES code directly in jmap extensions"

Reply:
A `jmap-extensions-elasticsearch` module under jmap folder alongside the modules `jmap-extensions` and `jmap-extensions-cassandra`. More clear? :)